flight/Engine.php

```diff
@@ -64,6 +64,7 @@
  * @method void etag(string $id, ('strong'|'weak') $type = 'strong') Handles ETag HTTP caching.
  * @method void lastModified(int $time) Handles last modified HTTP caching.
  */
+// phpcs:ignoreFile PSR2.Methods.MethodDeclaration.Underscore
```

Comment:
If we'll have phpcs to ignore rules? what's the sense of that?

Reply:
All I did was run composer phpcs and all those private methods that start with _ failed. This just ignores those errors for this file.